<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Boss</class>
 <widget class="QWidget" name="Boss">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>478</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageMain">
      <widget class="QPushButton" name="btnExit">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>370</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnView">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>270</y>
         <width>160</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>查看面试</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnModifyEN">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>截止提交申请</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>270</y>
         <width>160</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>安排面试官</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>231</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Curlz MT</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Boss Home</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageManage">
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>511</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>165</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>管理面试官</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>201</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>请输入查找的工号：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditFind">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnFind">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>50</y>
         <width>90</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>查找</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDelete">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>390</y>
         <width>110</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSure">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>390</y>
         <width>110</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBack1">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>390</y>
         <width>110</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAdd">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>110</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>增加</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
